### ISSUE TITLE
Release 12.0.0

### ISSUE THREAD

**Issue by voxpelli** [standard/eslint-config-standard#123]
As ESLint 5.0.0 is now released it would be neat to get eslint-config-standard 12.0.0 released as well.

Some stuff should probably be done first though:

Required:

* [ ] Change peerDependency to require the stable version
* [ ] (anything else?)

Nice to have:

* [x] Merge #108 – moves to newer `eslint-plugin-node`
* [x] Merge #126 – moves to even newer `eslint-plugin-node`
* [x] Merge #128 – moves to latest `eslint-plugin-promise`
* [x] Maybe merge #124 (see https://github.com/standard/eslint-config-standard/issues/123#issuecomment-401554891)
* [ ] (anything else?)

**Comment by StarpTech** [comment]
Does the PR consider also this issue when using eslint 5?

```
(node:12988) [ESLINT_LEGACY_OBJECT_REST_SPREAD] DeprecationWarning: The 'parserOptions.ecmaFeatures.experimentalObjectRestSpread' option is deprecated. Use 'parserOptions.ecmaVersion' instead. (found in "standard")
```

**Comment by yuchonghua** [comment]
![image](https://user-images.githubusercontent.com/17898715/42682742-005c0254-86be-11e8-928c-1dd5d6af0433.png)

**Comment by voxpelli** [comment]
@StarpTech @yuchonghua That should already be fixed in the alpha release of this module, see #117  and https://github.com/standard/eslint-config-standard/commit/6a89a51a2a0993e3e6f37f72eb4a461f754a593b, please try that one

**Comment by voxpelli** [comment]
@LinusU Added that to the list, is a review by @feross or such needed for that to be merged?

**Comment by yuchonghua** [comment]
When will it be released?
@voxpelli @LinusU

**Comment by yeziZ** [comment]
![678c45402d16ca06a16e8ef2ff57a25a](https://user-images.githubusercontent.com/26887147/42740333-a150a37e-88d8-11e8-981c-8099abaacf2d.jpg)

**Comment by voxpelli** [comment]
I guess that #126 should be merged as well before 12.0.0 is released, so that the module is as up to date as possible

**Comment by AllenZho** [comment]
It seems that all the upgrades with eslint5 are stuck here. Can it be solved as soon as possible? It doesn't have to be solved in the 12.0.0 release, you can solve it at 11.0.0+ first.
/ping @voxpelli @greenkeeper[bot]

**Comment by voxpelli** [comment]
@AllenZho See https://github.com/standard/eslint-config-standard/issues/123#issuecomment-404826773 – ESLint 5 is as far as I understand already supported by the released `12.0.0-alpha.0` and everyone who needs to be able to use ESLint 5 right here and now, and wants to do so without any warnings, can use that version.

Regarding when a stable `12.0.0` can be released – I don't know – I'm not a maintainer – I just opened this issue to help move it along. The only blocker from releasing a stable `12.0.0` is an action from a maintainer – all things outlined in this issue are just extra nice to haves that would be good, but not essential, to get included

**Comment by voxpelli** [comment]
@LinusU Any feedback on what/who we are waiting on to get this moving? Is it @feross that needs to find time for this or are there some topics that needs further feedback and/or work? And if so: Is there something one can do to help there

**Comment by LinusU** [comment]
From my side, I'm mostly waiting on @feross.

I would be very happy to move forward with this, but I'm not sure if we have the release process documented somewhere, and I'm not sure if I'm added on npm 🤔 

Would be great if you would want to give me some quick directions @feross ☺️ could be anything from "I'll have time soon, please wait" to "added you on npm, please go ahead" ❤️

**Comment by macklinu** [comment]
I just published [eslint-plugin-promise@v4](https://github.com/xjamundx/eslint-plugin-promise/blob/master/CHANGELOG.md#400), which supports ESLint v5 (drops support for Node v4). Not sure if that's on your list, but eslint-plugin-promise should no longer block this, if so. 👍

**Comment by voxpelli** [comment]
@macklinu Great! I added it to the list 👍

**Comment by feross** [comment]
Thank you to everyone on this issue! ❤️

I'm working to release `standard` 12 by tonight. Stay tuned!

(And I plan to documenting the release process and get a few others up to speed so I'm not a bottleneck in the future.)

**Comment by feross** [comment]
eslint-config-standard@12.0.0 is released!
